drivers/table.go

```diff
@@ -93,9 +92,7 @@ func (t Table) CanSoftDelete(deleteColumn string) bool {
 func TablesHaveNullableEnums(tables []Table) bool {
 	for _, table := range tables {
 		for _, col := range table.Columns {
-			if col.Nullable &&
-				(strings.HasPrefix(col.DBType, "enum.") || // postgresql
-					strings.HasPrefix(col.DBType, "enum(")) { // mysql
+			if col.Nullable && rgxEnum.MatchString(col.DBType) {
```

Comment:
You can use your freshly added `IsEnumDBType` here right? That way it is easier to refactor in the future if necessary?

Reply:
Yes, it can be used. Done.